terraform/modules/eks/main.tf

```diff
@@ -0,0 +1,169 @@
+data "aws_eks_cluster" "cluster" {
+  name = module.eks.cluster_id
+}
+
+data "aws_eks_cluster_auth" "cluster" {
+  name = module.eks.cluster_id
+}
+
+provider "kubernetes" {
+  host                   = data.aws_eks_cluster.cluster.endpoint
+  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
+  token                  = data.aws_eks_cluster_auth.cluster.token
+}
+
+module "eks" {
+  source          = "terraform-aws-modules/eks/aws"
+  version         = "18.10.1"
+  cluster_name    = var.cluster_name
+  cluster_version = "1.21"
+
+  subnet_ids = var.subnet_ids
+  vpc_id     = var.vpc_id
+
+  self_managed_node_group_defaults = {
+    instance_type                = "t2.small"
+  }
+
+  self_managed_node_groups = {
+    worker-group-1 = {
+      instance_type                 = "t2.small"
+      additional_security_group_ids = var.worker_sg_ids
+      min_size                      = 3
+      max_size                      = 8
+      desired_size                  = 3
+    }
+  }
+
+  # Extend cluster security group rules
+  cluster_security_group_additional_rules = {
+    egress_nodes_ephemeral_ports_tcp = {
+      description                = "To node 1025-65535"
+      protocol                   = "tcp"
+      from_port                  = 1025
+      to_port                    = 65535
+      type                       = "egress"
+      source_node_security_group = true
+    }
+  }
+
+  # Extend node-to-node security group rules
+  node_security_group_additional_rules = {
```

Comment:
Same here, should accept a list of additional security group rules to add. If we want we can keep these values as default and extend them, instead of just hardcode them as we did in the other modules